hydra/test_utils/test_utils.py

```diff
@@ -365,12 +365,13 @@ def run_python_script(
     env: Any = None,
     allow_warnings: bool = False,
     print_error: bool = True,
+    raise_exception: bool = True,
```

Comment:
Instead of adding an `raise_exception`, is it possible to  use `run_with_error` instead?
[example](https://github.com/facebookresearch/hydra/blob/c1ffab5357aeedfca3ef486dfae9f7e054e69588/tests/test_hydra_cli_errors.py#L68)

Reply:
Yes, thanks for the tip! Done in https://github.com/facebookresearch/hydra/pull/1616/commits/04b1cb0df98945276e23b5c39f78c649e2c66dbc.